### PR TITLE
Add angular 2 sample

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: templates/web-angular/web/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>A__projectName__Z</title>
+
+    <script async src="main.dart" type="application/dart"></script>
```
Comment (kwalrath) [inline templates/web-angular/web/index.html]: We recently agreed to change async -> defer everywhere

Comment (kevmoo) [inline templates/web-angular/web/index.html]: On both the app script and the browser/dart.js script?

**Comment by Sfshaza** [comment]
Yes, on both. On recommendation of Jake.

File: templates/web-angular/web/index.html (hunk 1 above)
Comment (kwalrath) [inline templates/web-angular/web/index.html]: Yep. You can see it here: https://github.com/angular/angular.io/pull/409/files#diff-dc5e06b1735b7fd31b98adf431e412fa

**Comment by devoncarew** [comment]
lgtm after rebase, but I'm not the best person to review an angular app -

**Comment by kwalrath** [comment]
lgtm

I haven't been using a library in main.dart, but if that's our style, I'm fine with it.

**Comment by kevmoo** [comment]
@kwalrath For small samples, putting things in `lib` is a bit overkill.

For stubbing out a new app, it's a great practice to put most code in `lib`. It makes it must easier to test components.

**Comment by kevmoo** [comment]
Closed with https://github.com/google/stagehand/commit/cc57dc1aa256c6b00fbc8ddb55df2b55f5c9655f

**Pull request closed without merging**